src/shared/GlobalTopBanners/GlobalTopBanners.tsx

```diff
@@ -8,18 +8,20 @@ const TrialBanner = lazy(() => import('./TrialBanner'))
 const TeamPlanFeedbackBanner = lazy(() => import('./TeamPlanFeedbackBanner'))
 const ProPlanFeedbackBanner = lazy(() => import('./ProPlanFeedbackBanner'))
 const OktaBanners = lazy(() => import('./OktaBanners'))
+const TokenlessBanner = lazy(() => import('./TokenlessBanner'))
 
 const GlobalTopBanners: React.FC = () => {
   return (
     <SilentNetworkErrorWrapper>
       {/* These are listed in priority order: if multiple banners are rendering, only the bottommost will display. */}
-      <div className="[&>*:last-child]:block [&>*]:hidden">
```

Comment:
was this removed on purpose?

Reply:
yes, it was overriding other banners instead of stacking them on top of each other.

Reply:
ooo, can we update the comment above too then? It sounds like that may have been the intended behavior at one point?

Reply:
good point!